src/components/useraccount/orderdetail/MainDetails.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div v-if="me">
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingAddress') }}</div>
+          <div>
+            {{me.order.shippingAddress.title}}
+            {{me.order.shippingAddress.firstName}}
+            {{me.order.shippingAddress.lastName}}
+          </div>
+          <div>{{me.order.shippingAddress.streetName}} {{me.order.shippingAddress.streetNumber}}</div>
+          <div>{{me.order.shippingAddress.additionalStreetInfo}}</div>
+          <div>{{me.order.shippingAddress.city}}</div>
+          <div>{{me.order.shippingAddress.postalCode}} {{me.order.shippingAddress.region}}</div>
+          <div>{{me.order.shippingAddress.country}}</div>
+          <br>
+          <div>{{me.order.shippingAddress.contactInfo.phone}}</div>
+          <div>{{me.order.shippingAddress.contactInfo.email}}</div>
+        </div>
+      </div>
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('billingAddress') }}</div>
+          <div>
+            {{me.order.billingAddress.title}}
```

Comment:
Since billing and shipping addresses are not mandatory fields in CTP (https://docs.commercetools.com/http-api-projects-orders#order) we always need to check if something exists before showing it, otherwise the page may break for some orders.